lib/sneakers/errors.rb

```diff
@@ -0,0 +1,5 @@
+require 'timeout'
+
+module Sneakers
+  class JobTimeout < Timeout::Error; end
```

Comment:
@dsbonev `Sneakers::Timeout` need to be fully qualified to not conflict with `::Timeout`. I don't feel particularly strongly about the name but fully-qualifying classes with clashing names is perfectly fine and a fact of life.